### PR TITLE
chore: Make nested styles import available for all libraries

### PR DESCRIPTION
**Pull request by bgambocjaviniar**
closes https://jira.tools.sap/browse/CXSPA-238

1. configuring custom webpack in order to alias and resolve to the proper library for import awareness
2. updated schematics for the shellapp to include node_modules to the styles preprocessor in order to access nested styles within packages

### CODE REVIEW

File: extra-webpack.config.js [hunk 1]
```diff
@@ -24,6 +24,37 @@ module.exports = {
...
+      ),
+      '@spartacus/storefinder': path.join(
+        __dirname,
+        'feature-libs/storefinder'
+      ),
+      '@spartacus/checkout': path.join(__dirname, 'feature-libs/checkout'),
+      '@spartacus/asm': path.join(__dirname, 'feature-libs/asm'),
+      '@spartacus/smartedit': path.join(__dirname, 'feature-libs/smartedit'),
+      '@spartacus/qualtrics': path.join(__dirname, 'feature-libs/qualtrics'),
+      '@spartacus/cdc': path.join(__dirname, 'feature-libs/cdc'),
```
Comment (Pucek9) [inline extra-webpack.config.js]:
```suggestion
      '@spartacus/cdc': path.join(__dirname, 'integration-libs/cdc'),
```

File: extra-webpack.config.js [hunk 2]
```diff
@@ -24,6 +24,37 @@ module.exports = {
...
+      '@spartacus/digital-payments': path.join(
+        __dirname,
+        'feature-libs/digital-payments'
```
Comment (Pucek9) [inline extra-webpack.config.js]:
```suggestion
        'integration-libs/digital-payments'
```

File: extra-webpack.config.js [hunk 3]
```diff
@@ -24,6 +24,37 @@ module.exports = {
...
+      ),
+      '@spartacus/tracking': path.join(__dirname, 'feature-libs/tracking'),
+      '@spartacus/cart': path.join(__dirname, 'feature-libs/cart'),
+      '@spartacus/order': path.join(__dirname, 'feature-libs/order'),
+      '@spartacus/setup': path.join(__dirname, 'feature-libs/setup'),
```
Comment (Pucek9) [inline extra-webpack.config.js]:
```suggestion
      '@spartacus/cds': path.join(__dirname, 'integration-libs/cds'),
```

File: extra-webpack.config.js (hunk 2 above)
Comment (bgambocjaviniar) [reply extra-webpack.config.js]: removed as there's no scss in that lib

File: extra-webpack.config.js (hunk 1 above)
Comment (bgambocjaviniar) [reply extra-webpack.config.js]: removed as there's no scss in that lib

File: extra-webpack.config.js (hunk 3 above)
Comment (bgambocjaviniar) [reply extra-webpack.config.js]: removed as there's no scss in that lib

Comment (Pucek9) [reply extra-webpack.config.js]:
```suggestion
```

**Review by Pucek9 — APPROVED** [review]

**Pull request merged**